src/features/lightspeed/playbookGeneration.ts

```diff
@@ -44,41 +42,46 @@ async function generatePlaybook(
     panel.webview.postMessage({ command: "exception" });
   }
 
+  const create_outline = false;
   const playbook: GenerationResponse = await client.sendRequest(
     "playbook/generation",
     {
       accessToken,
       URL: settingsManager.settings.lightSpeedService.URL,
-      content,
+      text,
+      outline,
+      create_outline,
```

Comment:
We can call the parameter `createOutline` since everything else is camelCase.

Reply:
Updated in the latest commit.